acai-ios/MenuItem.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  BowlItem.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/15/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import IGListKit
+
+class HeaderItem: ListDiffable {
```

Comment:
Can we make separate files for the models instead of putting them all into `MenuItem.swift`. will make it easier for finding files later